src/components/header/top-header/right-menu/constants.ts

```diff
@@ -2,18 +2,34 @@ import type { LinkItems } from '../../../../sources/types/types';
 import { PagePath } from '../../../../router/enums';
 import { messages } from './messages';
 import { IconType } from '../../../svg-builder/enums';
+import { userStore } from '../../../../store/user-store';
+import { computed } from 'mobx';
 
-const rightLinks: LinkItems[] = [
-  {
-    to: PagePath.loginPage,
-    label: messages.loginLink,
-    iconType: IconType.Login,
-  },
-  {
-    to: PagePath.registerPage,
-    label: messages.registerLink,
-    iconType: IconType.Registration,
-  },
-];
+const rightLinks = computed<LinkItems[]>(() => [
+  userStore.isAuth
+    ? {
+        to: PagePath.root,
+        label: messages.logoutLink,
+        iconType: IconType.LogOut,
+        onClick: () => userStore.logout(),
```

Comment:
```suggestion
        onClick: userStore.logout,
```
просто в сторе надо logout на стрелку переписать